transaction-status/src/lib.rs

```diff
@@ -250,7 +250,7 @@ impl EncodedTransaction {
                                     instruction,
                                     &transaction.message.account_keys,
                                 ) {
-                                    UiInstruction::Parsed(parsed_instruction)
+                                    UiInstruction::Parsed(json!(parsed_instruction))
```

Comment:
nit: could pass `ParsedInstruction` instead

Reply:
Had to add an additional enum to do this one. Let me know if you think it's not worth it.

Reply:
Ah, whoops! Looks fine